src/mimemail.erl

```diff
@@ -936,6 +932,11 @@ encode_quoted_printable(<<H, T/binary>>, Acc, L) ->
 	[A, B] = lists:flatten(io_lib:format("~2.16.0B", [H])),
 	encode_quoted_printable(T, [B, A, $= | Acc], L+3).
 
+insert_soft_newline([H | T], AfterPos) when AfterPos > 0 ->
+	[H | insert_soft_newline(T, AfterPos - 1)];
+insert_soft_newline(Str, 0) ->
+	[$\n, $\r, $= | Str].
```

Comment:
minor: I think you can do `"\n\r=" ++ Str` and compiler will be able to optimize it to what your code looks like now

Reply:
You are right! Should I change it before merge or keep it as it is?

Reply:
I'm fine as it is now.

Reply:
OK, thanks!